ansible_collections/arista/avd/roles/eos_designs/templates/base/snmp-server.j2

```diff
@@ -1,6 +1,11 @@
 {# snmp settings #}
 snmp_server:
 {% if snmp_settings is arista.avd.defined %}
+{%     if snmp_settings.compute_local_engineid is arista.avd.defined(true) %}
+{%         set local_engine_id = (inventory_hostname ~ switch.mgmt_ip) | hash('sha1') %}
```

Comment:
Please use `FQCN` on `hash`

Reply:
This is this built-in one: https://docs.ansible.com/ansible/latest/user_guide/playbooks_filters.html#hashing-and-encrypting-strings-and-passwords 
we are using it in other places without FQCN so probably need to open an issue:
https://github.com/aristanetworks/ansible-avd/blob/2252eb1c6c598a81f9f4b4ca1b21a9c7abab2f04/ansible_collections/arista/avd/roles/eos_designs/templates/connected_endpoints/port-channel-interfaces.j2#L84